### PR TITLE
Tried to correct grammar; light editing

### PR DESCRIPTION
**Pull request by CyanoTex**
This is my first pull request, so, I'd like to get some feedback on my changes and if they make sense. 😀

### CODE REVIEW

**Review by spirillen — APPROVED** [review]
It for sure make seance and looks great, just a shame you hit the wrong repository for adding your commit.

You should make all commits to https://mypdns.org/my-privacy-dns/matrix as this is a mirror (Read backup :wink:)

I will try to see if I can merge this one, but please use mypdns.org for the future commits :+1:

**Comment by spirillen** [comment]
Merged in https://mypdns.org/my-privacy-dns/matrix/-/commit/c29abce61c2587d768232275e9162df60ea834d9

Thanks a lot for your contribution, always nice to get whatever help you can get

**Comment by spirillen** [comment]
Thanks for helping me spell :+1:

**Comment by spirillen** [comment]
IF you care, could it be at your interest to make a few lines in the `readme.md` about why we don't whitelist and it is up to the individual users to keep there own whitelist, as we should not become judges of what kind of bad domains that should be allowed access to the individual users network, and that through the issue board are trying to do the best for giving people the right information, for them to decide for themself if they would allow some domains access despite they marked as bad.

**Comment by spirillen** [comment]
> I think this should be good, though, I'm having trouble with this part of your message:
"as we should not become judges of what kind of bad domains that should be allowed access to the individual users network,"

Many other lists will remove (whitelist) bad-domains if it brakes some third party site; I.e. google-* because without allowing this spyware, site x is "broken". By whitelisting a badware domain to be allowed on site x, they have now judge everyone else to become spied on, typically against the end users knowledge while they believe software y protected them.  We do not do this in any ways shape or form, that alone is the individuals choice who should be allowed on there network.

Basically it is mostly a comment or the argument for why we do not remove badware from the list, and it is solely up to the individual to decide this.

**Comment by CyanoTex** [comment]
> > I think this should be good, though, I'm having trouble with this part of your message:
> > "as we should not become judges of what kind of bad domains that should be allowed access to the individual users network,"
> 
> Many other lists will remove (whitelist) bad-domains if it brakes some third party site; I.e. google-* because without allowing this spyware, site x is "broken". By whitelisting a badware domain to be allowed on site x, they have now judge everyone else to become spied on, typically against the end users knowledge while they believe software y protected them. We do not do this in any ways shape or form, that alone is the individuals choice who should be allowed on there network.
> 
> Basically it is mostly a comment or the argument for why we do not remove badware from the list, and it is solely up to the individual to decide this.

"Q: Why don't you whitelist [domain].xyz?

A: 
1) It's up to individual users (i.e you) to create and maintain their own whitelist.

2) By whitelisting bad domain X because website Y uses bad domain X, we're deceiving our users who believe they're protected against bad domain X.

3) Through this repository's issue board, we're trying our best to give people the right information for them to decide whether or not they should allow access to some domains, despite them being marked as bad."

What do you think?

**Comment by spirillen** [comment]
I think I bow in gratitude...  Damned that's well written, just add it to a MR and it is in :100: :1st_place_medal:
